packages/dds/tree/src/shared-tree/publicContracts.ts

```diff
@@ -20,5 +21,5 @@ export const SharedTreeFactoryType = "https://graph.microsoft.com/types/tree";
 export const SharedTreeAttributes: IChannelAttributes = {
 	type: SharedTreeFactoryType,
 	snapshotFormatVersion: "0.0.0",
-	packageVersion: "0.0.0",
+	packageVersion: pkgVersion,
```

Comment:
Are there any compatability implications with this change?

Reply:
my understanding is that none of these versions are used for anything except debugging so it should not impact compat in any way.